tests/test_docs_request_bodies.py

```diff
@@ -571,12 +571,16 @@ async def handler(request, body: Dict):
                         properties:
                           id:
                             type: integer
+                          rank:
```

Comment:
You should not touch anything that not related to your functionality. This function tests `oneOf`, your change is about `allOf`

Reply:
oh. sorry